Can't install slipshow with opam

Tested using `opam switch 5.1.1` and `opam switch 4.14.2` on macos Sonoma 14.2:

```
opam pin slipshow git@github.com/panglesd/slipshow.git 
[ERROR] Could not synchronize
        /Users/cosmo/.opam/5.1.1/.opam-switch/sources/slipshow from
        "git+file:///Users/cosmo/Documents/projects/bbchallenge/bbslides/git@github.com/panglesd/slipshow.git":
        "/usr/bin/git fetch -q" exited with code 128
[slipshow.0.0.33] fetching sources failed: git+file:///Users/cosmo/Documents/projects/bbchallenge/bbslides/git@github.com/panglesd/slipshow.git
[ERROR] Error getting source from
        git+file:///Users/cosmo/Documents/projects/bbchallenge/bbslides/git@github.com/panglesd/slipshow.git:
          -
        git+file:///Users/cosmo/Documents/projects/bbchallenge/bbslides/git@github.com/panglesd/slipshow.git
```

--- a/README.md
+++ b/README.md
@@ -48,11 +48,11 @@ For a super quick start, just click visit [sliphub.choum.net](https://sliphub.ch
 
 For using slipshow offline, you'll need to install the slipshow tool. Currently, there are several ways to get it. The best is to download the precompiled binary from the [latest release](https://github.com/panglesd/slipshow/releases/latest).
 
-Slipshow is also package in some package manager:
+Slipshow is also packaged as an `opam` package:
 
 ```shell
-# For opam users. TODO: Update once it has been released to opam.
-$ opam pin slipshow git@github.com/panglesd/slipshow.git
+# For opam users.
+$ opam install slipshow
 ```
 
 Then, you can use the `slipshow` binary to compile a `.md` file to a slipshow presentation:

